tests/container_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+
+cd /project
+
+python -m venv /tmp/venv
+source /tmp/venv/bin/activate
```

Comment:
No we don't set the CWD in the container build. It will therefore default to the workspace root for vscode devcontainer and / for a simple launch.

In the container we set the venv to /venv and only install runtime dependencies.

Here we install dev dependencies into /tmp/venv

Reply:
But don't we want to use the system created `/venv` rather than making a new one?

Reply:
We could use the /venv but does it make any difference?

I was avoiding the complication of making an editable install of dev over the top of an existing runtime wheel install. 

By creating a separate venv for this we are verifying that the editable install works from scratch.

I'm not sure how important that is so would be OK with /venv or /tmp/venv

Reply:
If we use `/venv` rather than making a new one, then we're mirroring what we do with the dev container, so have a slight preference for using that